docs/quantization.tex

```diff
@@ -115,11 +115,56 @@ \subsection{Inicjalizacja książki kodowej}
   \item metoda rozdzielania (ang. splitting) 
 \end{enumerate}
 
-Pierwsza z wymienionych powyżej, czyli metoda losowa polega na wylosowaniu N wektorów, gdzie N jet liczba całkowitą. Oczywiście takie rozwiązanie jest odpowiednie, wtedy gdy nie mamy informacji na temat danych wektorowych. Posiadając informacje możemy wykorzystać ją do zawężenia obszaru losowego.
+Pierwsza z wymienionych powyżej, czyli metoda losowa polega na wylosowaniu \textit{N} wektorów, gdzie \textit{N} jet liczba całkowitą. Oczywiście takie rozwiązanie jest odpowiednie, wtedy gdy nie mamy informacji na temat danych wektorowych. Posiadając informacje możemy wykorzystać ją do zawężenia obszaru losowego.
```

Comment:
Kiedy chcesz stworzyć kursywę jak w równaniach matematycznych w tekście, to wystarczy objąć to dolarami, np:

```
Pierwsza z wymienionych powyżej, czyli metoda losowa polega na wylosowaniu $N$ wektorów, gdzie $N$
```

Ale nie zmieniaj tego już. Tak na przyszłość.